fix(icons): update size and add in icon

christiemolloy opened the pull request:
Fix #1149 

@rachael-phillips if this isn't ideal, I could also work on just having the icons show on mobile.

Also, @jschuler I can't get the "CopyIcon" to work, it breaks the build for some reason.

<img width="390" alt="Screen Shot 2019-06-04 at 10 01 42 AM" src="https://user-images.githubusercontent.com/20118816/58885298-f5a2d980-86af-11e9-86d7-6d0b092125cf.png">

<img width="772" alt="Screen Shot 2019-06-04 at 10 01 59 AM" src="https://user-images.githubusercontent.com/20118816/58885300-f5a2d980-86af-11e9-8c05-c398739d723f.png">
<img width="943" alt="Screen Shot 2019-06-04 at 10 02 04 AM" src="https://user-images.githubusercontent.com/20118816/58885301-f5a2d980-86af-11e9-873f-4a083b678125.png">

rachael-phillips commented:
Hi @christiemolloy I'm ok with keeping the icons there, but no matter what we'll need the text with the icon in mobile. We need more space between icon and the following text.

christiemolloy commented:
Lets hold off merging this until this issue is merged in React: https://github.com/patternfly/patternfly-react/issues/2145

christiemolloy commented:
@rachael-phillips do yo mind reviewing this again, I added some screenshots below at the different viewport sizes. Removed the icons for now as they weren't rendering for some reason, but we can always create an issue to add them later.

<img width="292" alt="Screen Shot 2019-07-15 at 10 01 14 AM" src="https://user-images.githubusercontent.com/20118816/61222159-27b04e00-a6e8-11e9-965f-a65b9e9a3cb9.png">
<img width="818" alt="Screen Shot 2019-07-15 at 10 02 03 AM" src="https://user-images.githubusercontent.com/20118816/61222160-27b04e00-a6e8-11e9-8357-a8f3a687a1f9.png">
<img width="491" alt="Screen Shot 2019-07-15 at 10 04 45 AM" src="https://user-images.githubusercontent.com/20118816/61222161-27b04e00-a6e8-11e9-8578-03f5130e6aa4.png">
